selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/theia/TheiaNewFileDialog.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.pageobject.theia;
+
+import static org.eclipse.che.selenium.pageobject.theia.TheiaNewFileDialog.Locators.CLOSE_ICON_XPATH;
+import static org.eclipse.che.selenium.pageobject.theia.TheiaNewFileDialog.Locators.INPUT_FIELD_XPATH;
+import static org.eclipse.che.selenium.pageobject.theia.TheiaNewFileDialog.Locators.OK_BUTTON_XPATH;
+import static org.eclipse.che.selenium.pageobject.theia.TheiaNewFileDialog.Locators.TITLE_XPATH;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import org.eclipse.che.selenium.core.webdriver.SeleniumWebDriverHelper;
+import org.openqa.selenium.By;
+
+@Singleton
+public class TheiaNewFileDialog {
+  private final SeleniumWebDriverHelper seleniumWebDriverHelper;
+
+  @Inject
+  private TheiaNewFileDialog(SeleniumWebDriverHelper seleniumWebDriverHelper) {
+    this.seleniumWebDriverHelper = seleniumWebDriverHelper;
+  }
+
+  public interface Locators {
+    String TITLE_XPATH = "//div[@class='dialogBlock']//div[text()='New File']";
+    String INPUT_FIELD_XPATH = "//div[@class='dialogBlock']//div[@class='dialogContent']/input";
```

Comment:
_input field_ sounds too general. What about `FILE_NAME_XPATH` or something like that?